core/templates/components/state-editor/state-content-editor/state-content-editor.component.html

```diff
@@ -1,7 +1,7 @@
 <!--
   Off-screen preview of the learner view card in order to check if the height of the card has exceeded a specified limit.
 -->
-<div class="oppia-shadow-preview-card" aria-hidden="true">
+<div class="oppia-shadow-preview-card" aria-hidden="false">
```

Comment:
This change seems wrong. Shouldn't the off-screen shadow card be hidden from ARIA readouts? It is hidden from a sighted user and we should preserve consistency here.

Reply:
@seanlip Initially lighthosue giving an issue so I was figuring out the solution, now I have fixed the error and also makde the changes accordingly. PTAL, thanks!